core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/views/LegacyYouTubePlayerView.kt

```diff
@@ -214,6 +221,11 @@ internal class LegacyYouTubePlayerView(context: Context, attrs: AttributeSet? =
     }
 
     fun getPlayerUiController(): PlayerUiController {
+        if (isUsingWebUi)
+            throw RuntimeException("You called 'getPlayerUiController()' on a player instance that is " +
+                    "using the web-based UI. You should set 'useNativeUi=\"true\"' in your xml view or " +
+                    "initialize your player object using 'initializeWithNativeUi' ")
+
```

Comment:
Change the error message to "To call 'getPlayerUiController' you need to enable native UI. But you are currently using web UI instead. To enable native UI set 'useNativeUi=\"true\"' in your xml view or initialize your player object using 'initializeWithNativeUi'"